packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -10,7 +10,7 @@
         :disabled="uploadOrFileCreationBlocked"
         class="oc-background-primary-gradient"
       >
-        <oc-icon name="add" color="#ffffff" />
+        <oc-icon name="add" variation="inverse" />
```

Comment:
The problem here is `variation="raw"` doesn't exist, if you give the oc-button `appearance="raw" variation="inverse"` you can omit the icon variation and the CSS below. I got a local commit, need to sync&push